docker/oidc-controller/Dockerfile.dev

```diff
@@ -13,4 +13,4 @@ ENV DOTNET_USE_POLLING_FILE_WATCHER 1
 
 WORKDIR /app
 
-ENTRYPOINT dotnet watch run --urls=http://+:5000
```

Comment:
Same as previous Dockerfile, leave as 5000 as se4rvices communicating on the internal Docker network will rely on this setting. The exposed port can/will be different (in docker-compose).